docs/development/olm-development.md

```diff
@@ -9,28 +9,25 @@ Additional steps need to be taken to ensure the operator can be deployed with OL
 ## Prerequisites
 
 * Ensure you have access to a Kubernetes cluster via `kubectl` with cluster admin permissions.
-* Install Go version 1.15 or higher.
+* Install Go version 1.17 or higher.
 * Install OLM on your cluster. This can be done using the `make install-olm` command.
 * Ability to push to a container registry that is accessible inside your Kubernetes cluster.
 
-## Step 1: Push the operator image to a registry
+## Step 1: Build and push the operator and bundle
 
-Run `make ko-publish IMAGE_REPO=<your-registry>`, pushing to a container registry that is accessible inside your Kubernetes cluster.
-Using `ko.local` or `kind.local` is not recommended, as this will not push the resulting image to a container registry.
+Run `make bundle-push IMAGE_REPO=<your-registry>`, pushing to a container registry that is
+accessible inside your Kubernetes cluster.
+This make command will push the operator and [operator bundle](https://olm.operatorframework.io/docs/tasks/creating-operator-bundle/)
+to the container registry.
+An operator bundle is an OCI artifact that tells OLM how to deploy your operator.
 
+Using `ko.local` or `kind.local` for `IMAGE_REPO` is not recommended, as this will not push the
+resulting images to an OCI-compliant container registry.
```

Comment:
That's an interesting behavior, @adambkaplan. I find the `ko` approach to `kind.local` registries very useful, but for OLM we really need OCI compliance.

Reply:
Yeah, found this the hard way when building the CI infra for the operator in the first place. Deploying an operator on OLM requires at minimum 3 container images:

1. The operator binary
2. The operator "bundle"
3. The catalog source

Item 3 needs to be pullable from a container registry. I _think_ item 2 must also be pullable from a container registry. In terms of dev/testing, pushing everything to a "real" registry just makes life easier.